integration-app/src/main/resources/application.properties

```diff
@@ -45,20 +45,31 @@ management.endpoints.web.exposure.include=health,info,prometheus,metrics
 # ccs.esourcing.ipallow-list= # comma separated list of ipv4 or ipv6 addresses from which requests are allowed from, empty list then no filtering
 # ccs.esourcing.actuator-ipallow-list= # comma separated list of ipv4 or ipv6 addresses from which actuator requests are allowed from, empty list then no filtering
 ccs.esourcing.actuator-ipallow-list=10.0.0.0/8
-ccs.esourcing.api-key-header=api-key
+# ccs.esourcing.api-key-header=
 # ccs.esourcing.api-keys= # comma separated list of API keys used for "authenticating" requests
 
 # ----------------------------------------
 # JAGGAER CLIENT API PROPERTIES
 # ----------------------------------------
-spring.security.oauth2.client.provider.jaggaer.token-uri=
-spring.security.oauth2.client.registration.jaggaer.client-id=
-spring.security.oauth2.client.registration.jaggaer.client-secret=
-spring.security.oauth2.client.registration.jaggaer.authorization-grant-type=
+# Grant type is actually hard coded within the app, however Spring requires that this property be set
+# as we've overridden some of the Spring Oauth configuration for the non-standard Jaggaer token response
+spring.security.oauth2.client.registration.jaggaer.authorization-grant-type=client_credentials
 ccs.esourcing.jaggaer.default.buyer-company-id=51435

```

Comment:
We should remove these - these shouldn't be in a public git repo.

Reply:
Just the buyer company id?